spec/PasswordPolicy.spec.js

```diff
@@ -757,4 +759,233 @@ describe("Password Policy: ", () => {
     });
   });
 
+  it('should fail if passwordPolicy.daysBeforeExpiry is not a number', done => {
+    reconfigureServer({
+      appName: 'passwordPolicy',
+      passwordPolicy: {
+        daysBeforeExpiry: "not a number"
+      },
+      publicServerURL: "http://localhost:8378/1"
+    }).then(() => {
+      fail('passwordPolicy.daysBeforeExpiry "not a number" test failed');
+      done();
+    }).catch(err => {
+      expect(err).toEqual('passwordPolicy.daysBeforeExpiry must be a positive number');
+      done();
+    });
+  });
+
+  it('should fail if passwordPolicy.daysBeforeExpiry is a negative number', done => {
+    reconfigureServer({
+      appName: 'passwordPolicy',
+      passwordPolicy: {
+        daysBeforeExpiry: -100
+      },
+      publicServerURL: "http://localhost:8378/1"
+    }).then(() => {
+      fail('passwordPolicy.daysBeforeExpiry negative number test failed');
+      done();
+    }).catch(err => {
+      expect(err).toEqual('passwordPolicy.daysBeforeExpiry must be a positive number');
+      done();
+    });
+  });
+
+  it('should succeed if logged in before password expires', (done) => {
+    const user = new Parse.User();
+    reconfigureServer({
+      appName: 'passwordPolicy',
+      passwordPolicy: {
+        daysBeforeExpiry: 1 // 1 day
+      },
+      publicServerURL: "http://localhost:8378/1"
+    }).then(() => {
+      user.setUsername("user1");
+      user.setPassword("user1");
+      user.set('email', 'user1@parse.com');
+      user.signUp().then((u) => {
+        Parse.User.logIn("user1", "user1").then((user) => {
+          done();
+        }).catch((error) => {
+          jfail(error);
+          fail('Login should have succeeded before password expiry.');
+          done();
+        });
+      }).catch((error) => {
+        jfail(error);
+        fail('Signup failed.');
+        done();
+      });
+    })
+  });
+
+  it('should fail if logged in after password expires', (done) => {
+    const user = new Parse.User();
+    reconfigureServer({
+      appName: 'passwordPolicy',
+      passwordPolicy: {
+        daysBeforeExpiry: 0.000005787 // 0.5 sec
```

Comment:
@bhaskaryasa 
another small nit...

can you pls use 
`5/(24*60*60)`
instead of
`0.000005787`